drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/export.rb

```diff
@@ -0,0 +1,79 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvImporter::Importer
+  class Export < GrdaWarehouse::Hud::Base
+    include ::HMIS::Structure::Export
+    include ImportConcern
+
+    # Because GrdaWarehouse::Hud::* defines the table name, we can't use table_name_prefix :(
+    self.table_name = 'hmis_2022_exports'
```

Comment:
All of these are going to be unable to show themselves in the future if we update the table name.  We need some shells to represent each version of the spec for digging around in the console and to support historic import log viewing.